src/js/cagov/tabs.js

```diff
@@ -4,80 +4,79 @@
 
 (function () {
  // Get relevant elements and collections
- var tabbed = document.querySelector('.tabs');
+ const tabbed = document.querySelectorAll('.tabs');
```

Comment:
Switching to `querySelectorAll` will remove the need to check for nulls; if nothing is found it will return an empty Nodelist.

Reply:
@carterm removed the checker [a65a718](https://github.com/Office-of-Digital-Services/California-State-Web-Template-Website/pull/1226/commits/a65a718fba41602c43d63968ce1f081fd2503f8f)